public/javascripts/index.js

```diff
@@ -159,11 +198,50 @@ function deleteResources(url, successCallback, errorCallback) {
     accepts: {
       text: "application/hal+json"
     },
-    success: function(data, textStatus, jQxhr) {
+    success: function() {
       successCallback();
     },
-    error: function(jqXhr, textStatus, errorThrown) {
+    error: function(jqXhr) {
       errorCallback(jqXhr.responseJSON);
     }
   });
 }
+
+function buildMaterialMenuItems(clickedElementData) {
+  const baseOptions = [
+    {
+      type: "normal",
+      text: "Delete pacts ...",
+      click: handleDeletePactsSelected
+    },
+    {
+      type: "normal",
+      text: "Delete integration...",
+      click: handleDeleteIntegrationsSelected
+    }
+  ];
+
+  const providerName = clickedElementData.providerName;
+  const consumerName = clickedElementData.consumerName;
+  const taggedPactUrls = clickedElementData.taggedPactUrls;
+  const taggedPactsOptions = taggedPactUrls.map(taggedPactUrl => {
+    const pactTagName = getTagNameFromTaggedPactUrl(taggedPactUrl);
+    return {
+      type: "normal",
+      text: `Delete pact tagged ${pactTagName}...`,
+      click: handleDeleteTagSelected({
```

Comment:
The text for the option and the confirmation are here https://github.com/pact-foundation/pact_broker/issues/467

Reply:
Fixed and add `...` to keep it consistent with others items
![image](https://user-images.githubusercontent.com/4962863/128633296-72a696f1-59c6-416f-a48d-e5e10c453e49.png)